docs/variables/customapi.md

```diff
@@ -4,41 +4,61 @@ id: customapi
-|     `x-fossabot-channelid`      |                        Internal channel ID of fossaobt user.                        |
-| `x-fossabot-channeldisplayname` |      The uppercased version, or internationalized version of the channel name       |
-|    `x-fossabot-channellogin`    |                 The lowercase channel name (also known as "login")                  |
-|    `x-fossabot-channelslug`     |         The **Fossabot channel URL**. Useful for linking to commands pages.         |
-|  `x-fossabot-channelprovider`   |                     The platform it was sent from, eg. `twitch`                     |
-| `x-fossabot-channelproviderid`  |                       The platforms channel ID, ie: Twitch ID                       |
-|     `x-fossabot-hasmessage`     | `true` or `false` depending on whether a message sent this (`false` if timer, etc). |
-|          `user-agent`           |                                `Fossabot Web Proxy`                                 |
+### Developers
 
-## Headers for Messages
+**All status codes** will be **ignored** and only the **raw body content** of the requested URL will be returned.
 
-These headers are for requests sent from messages, therefore are attached to a particular user.
+Fossabot also sends multiple custom HTTP headers that you can use to build deeper integrations, a list of them can be found below.
+
+### IP Ranges
+
+Fossabot's proxy **does not** use a consistent IP range to send requests. You **cannot** rely on it for authenticating endpoints!
+
+We recommended that you include a key query parameter or something else to authenticate users.
+
+**For that reason we do NOT print customapi URLs on public commands pages.**
+
+### List of Common Headers
+
+These headers are present in **all** requests.
+
+|              Name               |                                        Value                                         |
+| :-----------------------------: | :----------------------------------------------------------------------------------: |
+|     `x-fossabot-channelid`      |                   The internal **Fossabot ID** of a broadcaster.                     |
+| `x-fossabot-channeldisplayname` |   The uppercase version, or internationalized version of a broadcaster's username.   |
+|    `x-fossabot-channellogin`    |   The lowercase version of a broadcaster's username (otherwise known as *login*).    |
+|    `x-fossabot-channelslug`     | The **Fossabot channel URL** of a broadcaster. Useful for linking to commands pages. |
+|  `x-fossabot-channelprovider`   |               The platform the request was sent from (e.g. `twitch`).                |
+| `x-fossabot-channelproviderid`  |                The platform ID (e.g. the Twitch ID) of a broadcaster.                |
+|     `x-fossabot-hasmessage`     | `true` or `false` depending on whether a message sent this (`false` if timer, etc).  |
```

Comment:
maybe `message triggered this`, instead? (this is prob my typo, sorry)

Reply:
I'll change it to `depending on whether a message triggered this request`.